temporal_navigation/ObservedObject[]  semantic_objs
int32 day
---

